### DISCUSSION TITLE
Why is this panel free? چرا شما این پنل را رایگان منتشر می‌کنید؟

### DISCUSSION THREAD

**Discussion by TheTraxex** [hiddify/Hiddify-Manager#111] [Q&A سوالات رایج] [answered]
چرا شما این پنل را رایگان منتشر می‌کنید؟

**Comment by TheTraxex** [comment, marked as answer]
هدف از راه‌اندازی این پنل و تمامی فعالیت‌های مرتبط با آن که همگی به صورت داوطلبانه انجام میشه، دسترسی مردم ایران به اینترنت آزاد و بدون فیلتر است. تمامی این فعالیت‌های از روزهای آغازین فیلترینگ گسترده اینترنت در ایران پس از قتل مهسا امینی شروع شد و تا به حال ادامه دارد.

**Reply by ezrbidoky** [reply]
سلام بهتره بنویسید
.... پس از "قتل" مهسا امینی شروع ش.....


ممنون از زحماتتون

**Reply by hiddify-com** [reply]
اصلاح شد
